tests/unit/mpa/modules/models/scalar_schedulers/test_schedulers.py

```diff
@@ -0,0 +1,126 @@
+"""Test schedulers."""
+
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+
+from otx.mpa.modules.models.scalar_schedulers import (
+    ConstantScalarScheduler,
+    PolyScalarScheduler,
+    StepScalarScheduler,
+)
+
+
+class TestSchedulers:
+    """Test schedulers."""
+
+    def test_constant_scalar_scheduler(self):
+        """Test constant scalar scheduler."""
+        scheduler = ConstantScalarScheduler(scale=30.0)
+
+        # learning rate should not change over time
+        assert scheduler(0, 1) == 30.0
+        assert scheduler(1, 1) == 30.0
+        assert scheduler(2, 10) == 30.0
+
+    def test_constant_scalar_scheduler_invalid_scale(self):
+        """Test constant scalar scheduler with invalid scale."""
+        with pytest.raises(AssertionError):
+            ConstantScalarScheduler(scale=-1.0)
+
+    def test_constant_scalar_scheduler_invalid_step(self):
+        """Test constant scalar scheduler with invalid step.
+
+        TODO: ConstantScalarScheculer should be modified to raise this error
+        """
+        # scheduler = ConstantScalarScheduler(scale=30.0)
+        # with pytest.raises(AssertionError):
+        #     scheduler(-1, 1)
```

Comment:
Is this going to be a follow-up PR?

Reply:
Maybe we could use `xfail(reason)` to mark this test

Reply:
I think the scheduler should check for this but not sure if this will be implemented. I wanted to start a discussion on it. But xfail makes sense.